docs/porposals/diamond.md

```diff
@@ -0,0 +1,40 @@
+# Diamond workflow proposal.
+
+Diamond workflow allows for a pipeline to scale out the generation of a single bundle. There 
+is little or no co-ordination between the nodes. Each node uploads part of the bundle in the
+ filesystem namespace of the bundle. 
+
+The diamond workflow consists of the following steps.
+
+1. Before a diamond starts, Datamon SDK allows for the creation of an unique ID for a diamond.
```

Comment:
My assumption is that the unique ID is created by Datamon and returned (i.e. printed to stdout) by the CLI?

Reply:
@galvare2 yes it will be an ID to STDOUT at the end.